domain-model/src/main/java/uk/gov/hmcts/cmc/domain/constraints/PostcodeConstraintValidator.java

```diff
@@ -23,6 +21,6 @@ public boolean isValid(String value, ConstraintValidatorContext context) {
             return true;
         }
         String normalised = value.replaceAll(" ", "");
-        return pattern.matcher(normalised).matches();
+        return Pattern.matches(PATTERN, normalised);
```

Comment:
sorry looks like its better to have the Pattern statically compiled:

https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html#matches-java.lang.String-java.lang.CharSequence-

Reply:
done